src/strings.ts

```diff
@@ -0,0 +1,2 @@
+export const TALKER_TYPE: string = 'application/x-talkerjs-v1+json';
+export const TALKER_ERR_TIMEOUT: string = 'Talker.js message timed out waiting for a response.';
```

Comment:
Super nit: I kinda thought this was an actual error when I saw the variable in the other file. Maybe stick a `MSG` in there somewhere? `TALKER_MSG_TIMEOUT` or `TALKER_ERR_MSG_TIMEOUT`. Or even just `ERR_MSG_TIMEOUT`.